build-mingw.bat

```diff
@@ -0,0 +1,101 @@
+@echo off
+setlocal
+
+:: Adjust path to your 64-bit MinGW-w64 bin directory
+set PATH=C:\ProgramData\mingw64\mingw64\bin;%PATH%
+
+:: Output folders - create mingwbuild folder
+set OUT=mingwbuild
+if not exist %OUT% mkdir %OUT%
+
+:: LSL lib and include paths
+set LSL_INC="C:\Users\Isaiah\Documents\lsl-wearablesensing-final\liblsl-1.16.2\include"
```

Comment:
remove you full paths, feel free to use the relative and then ask users to add their full one